src/main/java/teammates/ui/controller/InstructorCourseStudentDetailsEditSaveAction.java

```diff
@@ -57,14 +57,25 @@ public ActionResult execute() throws EntityDoesNotExistException {
             
             boolean isSectionChanged = student.isSectionChanged(originalStudentAttribute);
             boolean isTeamChanged = student.isTeamChanged(originalStudentAttribute);
+            boolean isEmailChanged = student.isEmailChanged(originalStudentAttribute);
             if (isSectionChanged) {
                 logic.validateSectionsAndTeams(Arrays.asList(student), courseId);
             } else if (isTeamChanged) {
                 logic.validateTeams(Arrays.asList(student), courseId);
             }
             
             logic.updateStudent(studentEmail, student);
-            statusToUser.add(new StatusMessage(Const.StatusMessages.STUDENT_EDITED, StatusMessageColor.SUCCESS));
+            
+            boolean isSendEmail = getRequestParamAsBoolean(Const.ParamsNames.EMAIL_SEND_CHECK);
+            if (isEmailChanged && isSendEmail) {
+                logic.sendFeedbackSessionsSummaryOfCourseToNewStudentEmail(courseId, student);
+            }
+            if (isSendEmail) {
+                statusToUser.add(new StatusMessage(Const.StatusMessages.STUDENT_EDITED_AND_EMAIL_SENT,
+                                                   StatusMessageColor.SUCCESS));
+            } else {
+                statusToUser.add(new StatusMessage(Const.StatusMessages.STUDENT_EDITED, StatusMessageColor.SUCCESS));
+            }
```

Comment:
This if-else block can be simplified:
```java
statusToUser.add(new StatusMessage(isSendEmail ? ... : ... , StatusMessageColor.SUCCESS));
```
If the line becomes too long you can separate the `StatusMessage` variable out, or any other methods you can think of.